test/newton_neural_ode.jl

```diff
@@ -39,11 +39,11 @@ nODE = NeuralODE(NN, tspan, ROCK2(), reltol=1e-4, saveat=[tspan[end]])
 
 loss_function(θ) = Flux.mse(y, nODE(x, θ))
 l1 = loss_function(nODE.p)
-res = DiffEqFlux.sciml_train(loss_function, nODE.p, LBFGS(), GalacticOptim.AutoZygote(), maxiters = 400, cb=cb, allow_f_increases = true)
+res = DiffEqFlux.sciml_train(loss_function, nODE.p, LBFGS(), GalacticOptim.AutoZygote(), maxiters = 1000, cb=cb)
 @test 10loss_function(res.minimizer) < l1
-res = DiffEqFlux.sciml_train(loss_function, nODE.p, NewtonTrustRegion(), GalacticOptim.AutoZygote(), maxiters = 400, cb=cb, allow_f_increases = true)
+res = DiffEqFlux.sciml_train(loss_function, nODE.p, NewtonTrustRegion(), GalacticOptim.AutoReverseDiff(), maxiters = 400, cb=cb)

```

Comment:
That's an issue to solve. The Newton doesn't work with the adjoints anymore?

Reply:
It does, it's just very slow. I was surprised that changing the AD backend did it, It think the issue is in `FastChain`

Reply:
Actually it's an issue when using `ROCK2` with `FastChain` `ROCK4`

Reply:
And this test has been timing out on some runs from before https://travis-ci.com/github/SciML/DiffEqFlux.jl/builds/189598780